src/containers/Careers.js

```diff
@@ -1,16 +1,20 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
+import Card from '../components/Card';
+import LikeAndDislikeButtons from '../components/LikeAndDislikeButtons';
 
 const CareersContainer = styled.div`
-  color: blue;
+  background-color: white;
+  padding-top: 1%;
+  height: 90vh;
```

Comment:
I'll have a think about other ways to do this than 90/10vh. I think giving the navbar an arbitrary percent height just to keep the content filling the remainder is not a perfect solution.